README.md

```diff
@@ -65,9 +81,17 @@ Executing the systems of this service require different steps:
 > Press the green button next to the `main` method in `Session.java` and the server will be executed:
 ![executionImage](readmeresources/backendexecution.png)
 
+
+
+## How to report a bug
+No sure what to put on this section for now
```

Comment:
Direct them to making github issues for bug submissions, tell them to the the template

Reply:
Also say to add the `bug` label before submitting the issue